client/src/components/Shared/Form/index.js

```diff
@@ -13,15 +15,95 @@ import {
   StyledLabels,
   StyledInput,
   StyledTextarea,
-  StyledImgCon,
-}
-  from './form.style';
+  StyledImgCon
+} from "./form.style";
 
 class Form extends Component {
   constructor(props) {
     super(props);
+    this.state = {
+      isOpen: false,
```

Comment:
These all need to be in camel case to keep consistent with the rest of the project. Is there a reason you used snake case here?